recipes/melon/all/conanfile.py

```diff
@@ -0,0 +1,117 @@
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+class PackageConan(ConanFile):
+    name = "melon"
+    description = "A modern and efficient graph library using C++20 ranges and concepts."
+    # Use short name only, conform to SPDX License List: https://spdx.org/licenses/
+    # In case it's not listed there, use "LicenseRef-<license-file-name>"
+    license = "BSL-1.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/fhamonic/melon"
+    # Do not put "conan" nor the project name in topics. Use topics from the upstream listed on GH
+    # Keep 'header-only' as topic
+    topics = ("graph", "algorithm", "ranges", "c++20", "header-only")
+    package_type = "header-library"
+    # Keep these or explain why it's not required for this particular case
+    settings = "os", "arch", "compiler", "build_type"
+    # Do not copy sources to build folder for header only projects, unless you need to apply patches
+    no_copy_source = True
+
+    @property
+    def _min_cppstd(self):
+        return 20
+
+    # In case the project requires C++14/17/20/... the minimum compiler version should be listed
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "apple-clang": "14",
+            "clang": "17",
+            "gcc": "12",
+            "msvc": "192",
+            "Visual Studio": "17",
+        }
+
+    # Use the export_sources(self) method instead of the exports_sources attribute.
+    # This allows finer grain exportation of patches per version
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def layout(self):
+        # src_folder must use the same source folder name than the project
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        # Prefer self.requires method instead of requires attribute
+        # Direct dependencies of header only libs are always transitive since they are included in public headers
+        self.requires("range-v3/[>=0.11.0]", transitive_headers=True)
+        self.requires("fmt/[>=10.0.0]", transitive_headers=True)
+
+    # same package ID for any package
+    def package_id(self):
+        self.info.clear()
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            # Validate the minimum cpp standard supported when installing the package. For C++ projects only
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+            )
+
+        # In case this library does not work in some another configuration, it should be validated here too
+        if self.settings.os == "Windows":
```

Comment:
Is this a real condition? or did you forget to remove the example from the package template?

Reply:
You are right, I forgot..